public/js/notify.js

```diff
@@ -32,16 +32,26 @@ function setupNotification(callback) {
   }
 }
 
+const AUDIO = new Audio();
+// Maybe add the possibility for the notification to increase the sound's volume every n notifications?
+function playSound(fileName) {
+  AUDIO.src = fileName;
+  AUDIO.play();
+}
+
 /**
  * @param {string} notificationTitle Title
  * @param {{icon: string, body: string}} options Notification options
  */
 function notify(notificationTitle, options = {}) {
   setupNotification(() => {
     const notification = new Notification(notificationTitle, options);
+    const sound = `assets/${options.sound || 'blob'}.mp3`;
+    playSound(sound);
     notification.onclick = () => {
       // eslint-disable-next-line no-console
```

Comment:
suggestion to clean up this method by removing the log and commented alert

Reply:
Good suggestion :grin: and since in some browsers, clicking the notification redirect the user to the tab, I think it's safe to assume that the TODO underneath can be removed.